Modelplace_Cloud_API/modelplace_cloud_api.py

```diff
@@ -0,0 +1,55 @@
+from loguru import logger
+import time
+
+from api_requests import login_request, process_request, task_request 
+from utils import parse_args, save_prediction_result, save_visualization_result
+
+
+if __name__ == "__main__":
+    # Parse command line arguments
+    args = parse_args()
+
```

Comment:
`main.py` is supposed to be much simpler. Here you should call the function smth like this:

```python
args = parse_args()

access_token = get_access_token(email=args.email, password=args.password)
results = run_model(file, access_token=access_token, model_id=args.model_id)

save_results(results)
```